nucliadb/nucliadb/tests/test_search.py

```diff
@@ -952,3 +958,76 @@ async def test_only_search_calls_audit(nucliadb_reader, knowledgebox):
     assert resp.status_code == 200
 
     audit.search.assert_awaited_once()
+
+    resp = await nucliadb_reader.get(f"/kb/{kbid}/suggest?query=")
+
+    assert resp.status_code == 200
+
+    audit.suggest.assert_awaited_once()
+
+
+async def get_audit_messages(sub):
+    msg = await sub.fetch(1)
+    auditreq = AuditRequest()
+    auditreq.ParseFromString(msg[0].data)
+    return auditreq
+
+
+@pytest.mark.asyncio
+async def test_search_and_suggest_sent_audit(
+    nucliadb_reader,
+    knowledgebox,
+    stream_audit: StreamAuditStorage,
+):
+    from nucliadb_utils.settings import audit_settings
+
+    kbid = knowledgebox
+
+    # Prepare a test audit stream to receive our messages
+    partition = stream_audit.get_partition(kbid)
+    client: Client = await nats.connect(stream_audit.nats_servers)
+    jetstream: JetStreamContext = client.jetstream()
+    if audit_settings.audit_jetstream_target is None:
+        assert False, "Missing jetstream target in audit settings"
+    subject = audit_settings.audit_jetstream_target.format(
+        partition=partition, type="*"
+    )
+
+    set_utility(Utility.AUDIT, stream_audit)
+    try:
+        await jetstream.delete_stream(name=audit_settings.audit_stream)
+    except nats.js.errors.NotFoundError:
+        pass
+
+    await jetstream.add_stream(name=audit_settings.audit_stream, subjects=[subject])
+    psub = await jetstream.pull_subscribe(subject, "psub")
+
+    # Test search sends audit
+    resp = await nucliadb_reader.get(
+        f"/kb/{kbid}/search", headers={"x-ndb-client": "chrome_extension"}
+    )
+    assert resp.status_code == 200
+
+    auditreq = await get_audit_messages(psub)
+
+    assert auditreq.kbid == kbid
+    assert auditreq.type == AuditRequest.AuditType.SEARCH
+    assert (
+        auditreq.client_type == ClientType.CHROME_EXTENSION
+    )  # Just to use other that the enum default
+
+    # Test suggest sends audit
+    resp = await nucliadb_reader.get(
+        f"/kb/{kbid}/suggest?query=", headers={"x-ndb-client": "chrome_extension"}
+    )
+    assert resp.status_code == 200
+
+    auditreq = await get_audit_messages(psub)
+
+    assert auditreq.kbid == kbid
```

Comment:
To be more confident, you could have assertions for all attributes of `auditreq`

Reply:
from the POV of search, this are enough IMO, I agree that we may have unit tests on the `nucliadb_utils` for this that we don't right now, and we really need this to get rolling.